mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -345,6 +350,8 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		COMMENT
 		COMMENT <p>This is only called when the entity {@linkplain #getMoveEffect() has
 		COMMENT any move effect}, from {@link #move(MovementType, Vec3d)}
+	METHOD method_33574 setPosition (Lnet/minecraft/class_243;)V
```

Comment:
Just curious, what exactly does this one do? There is already a ton of set position methods for entity, some of which only set xyz and some changing pitch/yaw and recalculating stuff as well

Reply:
literally just calls the `setPosition(double x, double y, double z)` one using the values in the Vec3d